zbus_xmlgen/src/main.rs

```diff
@@ -140,6 +140,9 @@ fn write_interfaces(
     let mut process = match Command::new("rustfmt")
         .stdin(Stdio::piped())
         .stdout(Stdio::piped())
+        // rustfmt may post warnings about features not being enables on non-nightly rust
```

Comment:
* typo: enabled -> enabled.
* "stable" might be a better way to say "non-nightly". :wink: